lib/internal/buffer_old.js

```diff
@@ -11,6 +11,7 @@ const kMaxLength = smalloc.kMaxLength;
 exports.Buffer = Buffer;
 exports.SlowBuffer = SlowBuffer;
 exports.INSPECT_MAX_BYTES = 50;
+exports.kMaxLength = binding.kMaxLength;
```

Comment:
Ok then. I wanted to expose it as well but @bnoordhuis said it will probably become irrelevant soon: https://github.com/nodejs/io.js/pull/1822#issuecomment-106794588